src/vue/backend/src/store/modules/listingsSettings/actions.js

```diff
@@ -1,3 +1,7 @@
+const saveListingsSettings = ({ commit }, payload) => {
```

Comment:
This seems like it is a duplicate of saveGeneralListingsSettings

Reply:
I think there are four listings pages: Listings.vue, ListingsGeneral.vue, ListingsIdx.vue, and ListingsAdvanced.vue.
Listings.vue has the Enable switch.

Reply:
If you are making one just to ping the enable endpoint, then the name should be more specific to that. Something like `enable____`